javascript/ql/src/Security/CWE-601/examples/ServerSideUrlRedirectGood2.js

```diff
@@ -0,0 +1,21 @@
+const app = require("express")();
+
+function isLocalUrl(path) {
+  try {
+    return (
+      new URL(path, "https://example.com").origin === "https://example.com"
```

Comment:
I wonder if it would be a good idea to add a comment line above this with an instruction to replace `https://example.com` with your own domain name?